ui-kit/GroupCard/GroupCard.js

```diff
@@ -7,15 +7,15 @@ import Styled from './GroupCard.styles';
 import { textTrimmer } from '../../utils';
 
 const GroupCard = (props = {}) => {
-  const summaryLength = props.summary ? props.summary.length : 0;
+  const summaryLength = props?.summary?.length || 0;
   const maxChar = 130;
 
   // Option if we want to add a Avatar Count label for leaders
   // const heroAvatarsDiff = props.totalHeroAvatars - props.heroAvatars.length;
   const avatarsDiff = props.totalAvatars - props.avatars.length;
 
   return (
-    <Styled display="flex" flexDirection="column" {...props}>
+    <Styled.GroupCard {...props}>
```

Comment:
This will just be `<Styled>` once we make the changes discussed in `GroupCard.styles.js`.